charts/nri-kube-events/templates/_helpers_compatibility.tpl

```diff
@@ -3,8 +3,330 @@ Returns a dictionary with legacy runAsUser config.
+                {{ .Values.images.integration.registry }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.integration.repository" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.integration -}}
+            {{- if .Values.images.integration.repository -}}
+                {{ .Values.images.integration.repository }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.integration.tag" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.integration -}}
+            {{- if .Values.images.integration.tag -}}
+                {{ .Values.images.integration.tag }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.integration.pullPolicy" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.integration -}}
+            {{- if .Values.images.integration.pullPolicy -}}
+                {{ .Values.images.integration.pullPolicy }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.agent.registry" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.agent -}}
+            {{- if .Values.images.agent.registry -}}
+                {{ .Values.images.agent.registry }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.agent.repository" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.agent -}}
+            {{- if .Values.images.agent.repository -}}
+                {{ .Values.images.agent.repository }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.agent.tag" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.agent -}}
+            {{- if .Values.images.agent.tag -}}
+                {{ .Values.images.agent.tag }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- define "nri-kube-events.compatibility.new.agent.pullPolicy" -}}
+    {{- if .Values.images -}}
+        {{- if .Values.images.agent -}}
+            {{- if .Values.images.agent.pullPolicy -}}
+                {{ .Values.images.agent.pullPolicy }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+{{- /* Functions to fetch image configurations from globals */ -}}
+{{- define "nri-kube-events.compatibility.global.registry" -}}
+    {{- if .Values.global -}}
+        {{- if .Values.global.images -}}
+            {{- if .Values.global.images.registry -}}
+                {{ .Values.global.images.registry }}
+            {{- end -}}
+        {{- end -}}
+    {{- end -}}
+{{- end -}}
+
+
+
+{{/*
+Creates the image string needed to pull the integration image respecting the breaking change we made in the values file
```

Comment:
[nit] I would put this at the beginning, and add a small comment before all the function to retrieve the data

Reply:
Fixed in [#bd7e656](https://github.com/newrelic/nri-kube-events/pull/117/commits/bd7e65639f8c8c9fcfb6dfed470483c40260388b)